From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Doe <john.doe@somewhere.on.planet>
Date: Wed, 17 Jul 2024 14:16:12 +0000
Subject: Patching u-boot rk35xx files arch/arm/dts/rk3588s-nanopi-r6c.dts

Signed-off-by: John Doe <john.doe@somewhere.on.planet>
---
 arch/arm/dts/rk3588s-nanopi-r6c.dts | 97 ++++++++++
 1 file changed, 97 insertions(+)

diff --git a/arch/arm/dts/rk3588s-nanopi-r6c.dts b/arch/arm/dts/rk3588s-nanopi-r6c.dts
index 912b0a6533..e74aee5d9b 100644
--- a/arch/arm/dts/rk3588s-nanopi-r6c.dts
+++ b/arch/arm/dts/rk3588s-nanopi-r6c.dts
@@ -36,10 +36,107 @@
 		enable-active-high;
 		regulator-boot-on;
 	};
 };
 
+&cpu_l0 {
+	cpu-supply = <&vdd_cpu_lit_s0>;
+	mem-supply = <&vdd_cpu_lit_mem_s0>;
+};
+
+&cpu_b0 {
+	cpu-supply = <&vdd_cpu_big0_s0>;
+	mem-supply = <&vdd_cpu_big0_mem_s0>;
+};
+
+&cpu_b2 {
+	cpu-supply = <&vdd_cpu_big1_s0>;
+	mem-supply = <&vdd_cpu_big1_mem_s0>;
+};
+
+&i2c0 {
+    u-boot,dm-pre-reloc;
+	status = "okay";
+
+	vdd_cpu_big0_s0: vdd_cpu_big0_mem_s0: rk8602@42 {
+        u-boot,dm-pre-reloc;
+		compatible = "rockchip,rk8602";
+		reg = <0x42>;
+		vin-supply = <&vcc5v0_sys>;
+		regulator-compatible = "rk860x-reg";
+		regulator-name = "vdd_cpu_big0_s0";
+		regulator-min-microvolt = <550000>;
+		regulator-max-microvolt = <1050000>;
+		regulator-ramp-delay = <2300>;
+		rockchip,suspend-voltage-selector = <1>;
+		regulator-always-on;
+		regulator-boot-on;
+
+		regulator-state-mem {
+            u-boot,dm-pre-reloc;
+			regulator-off-in-suspend;
+		};
+	};
+
+	vdd_cpu_big1_s0: vdd_cpu_big1_mem_s0: rk8603@43 {	
+        u-boot,dm-pre-reloc;
+		compatible = "rockchip,rk8603";
+		reg = <0x43>;
+		vin-supply = <&vcc5v0_sys>;
+		regulator-compatible = "rk860x-reg";
+		regulator-name = "vdd_cpu_big1_s0";
+		regulator-min-microvolt = <550000>;
+		regulator-max-microvolt = <1050000>;
+		regulator-ramp-delay = <2300>;
+		rockchip,suspend-voltage-selector = <1>;
+		regulator-always-on;
+		regulator-boot-on;
+
+		regulator-state-mem {
+			u-boot,dm-pre-reloc;
+			regulator-off-in-suspend;
+		};
+	};
+};
+
+&i2c2 {
+	u-boot,dm-pre-reloc;
+	status = "okay";
+
+	vdd_npu_s0: vdd_npu_mem_s0: rk8602@42 {
+		u-boot,dm-pre-reloc;
+		compatible = "rockchip,rk8602";
+		reg = <0x42>;
+		vin-supply = <&vcc5v0_sys>;
+		regulator-compatible = "rk860x-reg";
+		regulator-name = "vdd_npu_s0";
+		regulator-min-microvolt = <550000>;
+		regulator-max-microvolt = <950000>;
+		regulator-ramp-delay = <2300>;
+		rockchip,suspend-voltage-selector = <1>;
+		regulator-always-on;
+		regulator-boot-on;
+
+		regulator-state-mem {
+			u-boot,dm-pre-reloc;
+			regulator-off-in-suspend;
+		};
+	};
+};
+
+&combphy2_psu {
+        u-boot,dm-pre-reloc;
+        status = "okay";
+};
+
+&pcie2x1l2 {
+	u-boot,dm-pre-reloc;
+	reset-gpios = <&gpio3 RK_PD1 GPIO_ACTIVE_HIGH>;
+	vpcie3v3-supply = <&vcc_3v3_pcie20>;
+	status = "okay";
+};
+
 &sdhci {
 	u-boot,dm-pre-reloc;
 	bus-width = <8>;
 	no-sdio;
 	no-sd;
-- 
Created with Armbian build tools https://github.com/armbian/build

