From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Doe <john.doe@somewhere.on.planet>
Date: Wed, 17 Jul 2024 14:23:28 +0000
Subject: Patching u-boot rk35xx files arch/arm/dts/rk3588s-nanopi-r6c.dts

Signed-off-by: John Doe <john.doe@somewhere.on.planet>
---
 arch/arm/dts/rk3588s-nanopi-r6c.dts | 15 ----------
 1 file changed, 15 deletions(-)

diff --git a/arch/arm/dts/rk3588s-nanopi-r6c.dts b/arch/arm/dts/rk3588s-nanopi-r6c.dts
index e74aee5d9b..25d8e825a9 100644
--- a/arch/arm/dts/rk3588s-nanopi-r6c.dts
+++ b/arch/arm/dts/rk3588s-nanopi-r6c.dts
@@ -36,25 +36,10 @@
 		enable-active-high;
 		regulator-boot-on;
 	};
 };
 
-&cpu_l0 {
-	cpu-supply = <&vdd_cpu_lit_s0>;
-	mem-supply = <&vdd_cpu_lit_mem_s0>;
-};
-
-&cpu_b0 {
-	cpu-supply = <&vdd_cpu_big0_s0>;
-	mem-supply = <&vdd_cpu_big0_mem_s0>;
-};
-
-&cpu_b2 {
-	cpu-supply = <&vdd_cpu_big1_s0>;
-	mem-supply = <&vdd_cpu_big1_mem_s0>;
-};
-
 &i2c0 {
     u-boot,dm-pre-reloc;
 	status = "okay";
 
 	vdd_cpu_big0_s0: vdd_cpu_big0_mem_s0: rk8602@42 {
-- 
Created with Armbian build tools https://github.com/armbian/build

