From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Doe <john.doe@somewhere.on.planet>
Date: Thu, 18 Jul 2024 10:27:37 +0000
Subject: Patching u-boot rk35xx files arch/arm/dts/rk3588s-nanopi-r6c.dts

Signed-off-by: John Doe <john.doe@somewhere.on.planet>
---
 arch/arm/dts/rk3588s-nanopi-r6c.dts | 131 ----------
 1 file changed, 131 deletions(-)

diff --git a/arch/arm/dts/rk3588s-nanopi-r6c.dts b/arch/arm/dts/rk3588s-nanopi-r6c.dts
index f30e02b356..5e6d2c9a3a 100644
--- a/arch/arm/dts/rk3588s-nanopi-r6c.dts
+++ b/arch/arm/dts/rk3588s-nanopi-r6c.dts
@@ -23,141 +23,10 @@
 		regulator-min-microvolt = <5000000>;
 		regulator-max-microvolt = <5000000>;
 		regulator-always-on;
 		regulator-boot-on;
 	};
-
-	vcc_3v3_sd_s0: vcc-3v3-sd-s0 {
-        u-boot,dm-pre-reloc;
-		compatible = "regulator-fixed";
-		regulator-name = "vcc_3v3_sd_s0";
-		gpio = <&gpio4 RK_PB4 GPIO_ACTIVE_HIGH>;
-		regulator-max-microvolt = <3000000>;
-		regulator-min-microvolt = <3000000>;
-		regulator-always-on;
-		enable-active-high;
-		regulator-boot-on;
-	};
-
-	vcc_3v3_pcie20: vcc3v3-pcie20 {
-        u-boot,dm-pre-reloc;
-		compatible = "regulator-fixed";
-		regulator-name = "vcc_3v3_pcie20";
-		regulator-min-microvolt = <3300000>;
-		regulator-max-microvolt = <3300000>;
-		regulator-always-on;
-		regulator-boot-on;
-	};
-};
-
-&i2c0 {
-    u-boot,dm-pre-reloc;
-	status = "okay";
-
-	vdd_cpu_big0_s0: vdd_cpu_big0_mem_s0: rk8602@42 {
-        u-boot,dm-pre-reloc;
-		compatible = "rockchip,rk8602";
-		reg = <0x42>;
-		vin-supply = <&vcc5v0_sys>;
-		regulator-compatible = "rk860x-reg";
-		regulator-name = "vdd_cpu_big0_s0";
-		regulator-min-microvolt = <550000>;
-		regulator-max-microvolt = <1050000>;
-		regulator-ramp-delay = <2300>;
-		rockchip,suspend-voltage-selector = <1>;
-		regulator-always-on;
-		regulator-boot-on;
-
-		regulator-state-mem {
-            u-boot,dm-pre-reloc;
-			regulator-off-in-suspend;
-		};
-	};
-
-	vdd_cpu_big1_s0: vdd_cpu_big1_mem_s0: rk8603@43 {	
-        u-boot,dm-pre-reloc;
-		compatible = "rockchip,rk8603";
-		reg = <0x43>;
-		vin-supply = <&vcc5v0_sys>;
-		regulator-compatible = "rk860x-reg";
-		regulator-name = "vdd_cpu_big1_s0";
-		regulator-min-microvolt = <550000>;
-		regulator-max-microvolt = <1050000>;
-		regulator-ramp-delay = <2300>;
-		rockchip,suspend-voltage-selector = <1>;
-		regulator-always-on;
-		regulator-boot-on;
-
-		regulator-state-mem {
-			u-boot,dm-pre-reloc;
-			regulator-off-in-suspend;
-		};
-	};
-};
-
-&i2c2 {
-	u-boot,dm-pre-reloc;
-	status = "okay";
-
-	vdd_npu_s0: vdd_npu_mem_s0: rk8602@42 {
-		u-boot,dm-pre-reloc;
-		compatible = "rockchip,rk8602";
-		reg = <0x42>;
-		vin-supply = <&vcc5v0_sys>;
-		regulator-compatible = "rk860x-reg";
-		regulator-name = "vdd_npu_s0";
-		regulator-min-microvolt = <550000>;
-		regulator-max-microvolt = <950000>;
-		regulator-ramp-delay = <2300>;
-		rockchip,suspend-voltage-selector = <1>;
-		regulator-always-on;
-		regulator-boot-on;
-
-		regulator-state-mem {
-			u-boot,dm-pre-reloc;
-			regulator-off-in-suspend;
-		};
-	};
-};
-
-&combphy2_psu {
-        u-boot,dm-pre-reloc;
-        status = "okay";
-};
-
-&pcie2x1l2 {
-	u-boot,dm-pre-reloc;
-	reset-gpios = <&gpio3 RK_PD1 GPIO_ACTIVE_HIGH>;
-	vpcie3v3-supply = <&vcc_3v3_pcie20>;
-	status = "okay";
-};
-
-&sdhci {
-	u-boot,dm-pre-reloc;
-	bus-width = <8>;
-	no-sdio;
-	no-sd;
-	non-removable;
-	max-frequency = <200000000>;
-	mmc-hs400-1_8v;
-	mmc-hs400-enhanced-strobe;
-	full-pwr-cycle-in-suspend;
-	status = "okay";
-};
-
-&sdmmc {
-	u-boot,dm-pre-reloc;
-	max-frequency = <150000000>;
-	no-sdio;
-	no-mmc;
-	bus-width = <4>;
-	cap-mmc-highspeed;
-	cap-sd-highspeed;
-	disable-wp;
-	sd-uhs-sdr104;
-	vmmc-supply = <&vcc_3v3_sd_s0>;
-	status = "okay";
 };
 
 &crypto {
 	status = "okay";
 };
-- 
Created with Armbian build tools https://github.com/armbian/build

